$OpenBSD: patch-darcs_cabal,v 1.10 2018/01/08 10:18:39 kili Exp $

Let it build with newer versions of libraries shipped with ghc-8.2.

Index: darcs.cabal
--- darcs.cabal.orig
+++ darcs.cabal
@@ -388,7 +388,7 @@ Library
       c-sources:      src/h_iconv.c
       build-depends:  unix >= 2.6.0.1 && < 2.8
 
-    build-depends:   base >= 4.8 && < 4.10,
+    build-depends:   base >= 4.8 && < 4.12,
                      binary >= 0.5 && < 0.9,
                      containers >= 0.5 && < 0.6,
                      regex-compat-tdfa >= 0.95.1 && < 0.96,
@@ -417,12 +417,12 @@ Library
                      unix-compat  >= 0.1.2 && < 0.5,
                      bytestring   >= 0.10.0.2 && < 0.11,
                      old-time     >= 1.1 && < 1.2,
-                     time         >= 1.5 && < 1.8,
+                     time         >= 1.5 && < 1.9,
     -- release notes of GHC 7.10.2 recommends to use text >= 1.2.1.3:
     -- https://mail.haskell.org/pipermail/haskell/2015-July/024641.html
                      text         >= 1.2.1.3 && < 1.3,
                      directory    >= 1.2.0.1 && < 1.4.0.0,
-                     process      >= 1.1.0.2 && < 1.5.0.0,
+                     process      >= 1.1.0.2 && < 1.7.0.0,
                      array        >= 0.4.0.1 && < 0.6,
                      random       >= 1.0.1.1 && < 1.2,
                      hashable     >= 1.1.2.5 && < 1.3,
@@ -529,7 +529,7 @@ Executable          darcs
   cc-options:       -D_REENTRANT
 
   build-depends:   darcs,
-                   base              >= 4.8 && < 4.10
+                   base              >= 4.8 && < 4.12
 
   -- if true to work around cabal bug with flag ordering
   if true
@@ -562,7 +562,7 @@ test-suite darcs-test
     build-depends:  Win32 >= 2.2 && < 2.4
 
   build-depends:   darcs,
-                   base         >= 4.8 && < 4.10,
+                   base         >= 4.8 && < 4.12,
                    array        >= 0.4.0.1 && < 0.6,
                    bytestring   >= 0.10.0.2 && < 0.11,
                    cmdargs      >= 0.10 && < 0.11,
